### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by Tahfimul**
#### Summary

Added pass by value changes value of array when passed to function.

#### Motivation

As another example to help understand the case of array passed as a value to a function and any change made to that array, impacts the array from a global scope perspective.

#### Supporting details

https://medium.com/@sayes2x/passing-an-array-to-a-function-as-a-value-in-javascript-10c20d5b89

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/guide/functions/index.md [hunk 1]
```diff
@@ -57,6 +57,22 @@ y = mycar.make; // y gets the value "Toyota"
...
 ```
 
+When you pass an array as a parameter, if the function changes any of the array's value, that change is visible outside the function, as shown in the following example:
+
+```js
+function myFunc(theArr) {
+  theArr[0] = 30;
+}
+
+var arr = [45];
```
Comment (OnkarRuikar) [inline files/en-us/web/javascript/guide/functions/index.md:67]:
```suggestion
const arr = [45];
```
Prefer `const` and `let`. [[code guidelines](https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Code_guidelines/JavaScript#declaring_variables)]

File: files/en-us/web/javascript/guide/functions/index.md [hunk 2]
```diff
@@ -57,6 +57,22 @@ y = mycar.make; // y gets the value "Toyota"
...
+var a, b;
```
Comment (OnkarRuikar) [inline files/en-us/web/javascript/guide/functions/index.md:68]:
```suggestion
let a, b;
```

File: files/en-us/web/javascript/guide/functions/index.md [hunk 3]
```diff
@@ -57,6 +57,22 @@ y = mycar.make; // y gets the value "Toyota"
                 // (the make property was changed by the function)
 ```
 
+When you pass an array as a parameter, if the function changes any of the array's value, that change is visible outside the function, as shown in the following example:
```
Comment (wbamberg) [inline files/en-us/web/javascript/guide/functions/index.md:60]:
```suggestion
When you pass an array as a parameter, if the function changes any of the array's values, that change is visible outside the function, as shown in the following example:
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @Tahfimul !

**Comment by Tahfimul** [comment]
No problem @wbamberg ! Sorry for the late reply.

**Pull request merged**